src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    this.constructor.alive.push(this);
+  }
+
+  static removeAnimal() {
```

Comment:
More appropriate naming for this method would be `removeDeadAnimals` or smth like that. `removeAnimal` name implies that you remove a single animal - which is not exactly correct.